website1/src/App.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import './App.css';
+import EllaID from './images/EllaID.jpg';
+import cooking from './images/cooking.png';
+import hiking from './images/IMG_9518.JPG';
+import skiing from './images/IMG_0031.JPG';
+import horses from './images/IMG_3842.jpg';
+import ridge from './images/IMG_9653.jpg';
+import packTrip from './images/IMG_9099.JPG';
+import lake from './images/IMG_9783.JPG';
+import ropes from './images/IMG_9687.JPG'
+
+function App() {
+  return (
+    <div className="App">
+      <head>
```

Comment:
sorry I never made this clear. `<head>` and `<body>` tags dont really go in the JS files in react. The `<head>` tag goes in the `index.html` file in the `public folder` (I commented where you can change the title). The style sheet is "linked" by importing it into the file (I left a comment above too). The `<body>` tag can just be replaced with a `<div>`.